libs/ui/src/lib/task-runner/task-runner.component.scss

```diff
@@ -29,22 +26,23 @@
   background-color: rgb(240, 240, 240);
 }
 
+.content-container {
+  height: calc(100% - 45px);
```

Comment:
Can you extract 45px to a descriptive variable called $WINDOW_CHROME_HEIGHT and use it here and below?

Reply:
Good point!